VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -190,29 +210,29 @@
                 *ngIf="!screenShareStream && canShowScreenShareButton" appTooltip
                 [text]="'private-consultation-room-controls.share-my-screen' | translate" colour="grey"
                 (click)="showEvidenceContextMenu = !showEvidenceContextMenu">
-                <span class="fak fa-start-screen-share"></span>
+                <fa-icon icon='share-square'></fa-icon>
 
                 <div id="shareDropdown" class="dropdown-content govuk-!-padding-0" [ngClass]="{ 'show': showEvidenceContextMenu }">
                     <div id="shareDropdown-startEvidenceSharing" *ngIf="enableDynamicEvidenceSharing && canShowDynamicEvidenceShareButton"
                         (click)="startScreenShareWithMicrophone()"
                         disableChildTooltip
                         appTooltip
                         [text]="'private-consultation-room-controls.share-video-evidence' | translate" colour="grey">
-                        <span class="action"><i class="fad fa-play-circle" aria-hidden="true"></i>
+                        <span class="action"><fa-icon [icon]="['far', 'play-circle']" aria-hidden="true"></fa-icon>
```

Comment:
Again uses far and the other used fad:
https://fontawesome.com/v5.15/how-to-use/on-the-web/referencing-icons/basic-use

Reply:
fad is not free for this icon. Although maybe fas would be more appropriate here. Let me ask Neil.